pages/11ty/url.filter.js

```diff
@@ -0,0 +1,7 @@
+const {github} = require('./site.config');
+
+const ghurl = (url) => github.srcUrl + url;
+
+module.exports = (config) => {
+  config.addFilter('ghurl', ghurl);
+};
```

Comment:
Please remove the filter if it's only used for string concatenation. We can handle that in the .njk file. Otherwise, ensure the use of appropriate tools to concatenate URLs internally, and make certain that any input URL is acceptable.